node_modules/oae-tenants/lib/rest.js

```diff
@@ -219,11 +219,13 @@ OAE.globalAdminRouter.on('post', '/api/tenant/create', function(req, res) {
  * @Server      admin
  * @Method      GET
  * @Path        /tenants
+ * @QueryParam  {number}        [start]         The index to start paging from
+ * @QueryParam  {number}        [limit]         The maximum number of results to return
  * @Return      {Tenants}                       All available tenants
  * @HttpResponse                200             Tenants available
  */
 OAE.globalAdminRouter.on('get', '/api/tenants', function(req, res, next) {
-    var tenants = TenantsAPI.getTenants();
+    var tenants = TenantsAPI.getTenants(false, req.query.start, req.query.limit);
```

Comment:
As we can't default the limit in the API, we should do that here:

var limit = OaeUtil.getNumberParam(req.query.limit, 10, 1);
var tenants = TenantsAPI.getTenants(false, req.query.start, limit);